### PR TITLE
clarifying the usage of coord and addresses in the stop event service

### PR DESCRIPTION
**Pull request by ue71603**
fixes https://github.com/VDVde/OJP/issues/407

### CODE REVIEW

**Review by herlitze — COMMENTED** [review]
I disagree with the next stop being always shown. If no stop is reached within range  of limitations , I would not expect a result.

File: OJP/OJP_StopEvents.xsd [hunk 1]
```diff
@@ -15,7 +15,7 @@
 		<xs:sequence>
 			<xs:element name="Location" type="PlaceContextStructure">
 				<xs:annotation>
-					<xs:documentation>LOCATION / PLACE for which to obtain stop event information.</xs:documentation>
+					<xs:documentation>LOCATION / PLACE for which to obtain stop event information. If a coordinate or an address is used, then the result may depend on other parameters: at least the next stop must be shown. All stops in reasonable walking distance and stops that are within range of limitations of the IndividualTransportAptions should be shown.</xs:documentation>
```
Comment (ue71603) [inline OJP/OJP_StopEvents.xsd:18]:
```suggestion
					<xs:documentation>LOCATION / PLACE for which to obtain stop event information. If a coordinate or an address is used, then the result may depend on other parameters: All stops in reasonable walking distance and stops that are within range of limitations of the IndividualTransportAptions should be shown. If the next stop is too far away, then no result is shown. "Reasonable" is usually defined as part of the server configuration.</xs:documentation>
```

Comment (ue71603) [reply OJP/OJP_StopEvents.xsd:18]: @herlitze  better?

**Review by herlitze — DISMISSED** [review]

File: OJP/OJP_StopEvents.xsd (hunk 1 above)
Comment (herlitze) [reply OJP/OJP_StopEvents.xsd:18]: Yes, thanks.

**Comment by skinkie** [comment]
@herlitze don't approve if you have open comments ;-)

**Comment by herlitze** [comment]
> @herlitze don't approve if you have open comments ;-)

Oh, you are right Matthias' suggested change is not part of the commit yet.

**Review by herlitze — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**